src/createElement.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import StemContainer from './StemContainer';
```

Comment:
I wish all of my things were as straightforward to name as `react-router-relay`, honestly. I hate naming things.